syft/serde/msgpack/serde.py

```diff
@@ -455,7 +455,7 @@ def _simplify(worker: AbstractWorker, obj: object, **kwargs) -> object:
 
 
 def _detail_field(typeCode, val):
-    if typeCode == 5 and val == "18446744073709551616":
+    if typeCode == 5 and val == str(2 ** 64):
```

Comment:
What does `typeCode == 5` signify here? Which type does `5` mean?

Reply:
It's the proto.json defined code for type str

Reply:
Do you need to hardcode it? Isn't there a way to have the code from the type?

Reply:
I haven't explored that possibility yet. I will look into it.

Reply:
I think you can use something [like this](https://github.com/OpenMined/PySyft/blob/ba804732961722d9e9b41e19fd580691bf5ebf42/test/serde/msgpack/test_msgpack_serde.py#L51).

Reply:
Done :+1: